File: 99_bottles.py
def bottle_song(num):
	for i in range(num,-1,-1):
		if i > 1:
			print(f'{i} bottles of beer on the wall, {i} bottles of beer. Take one down and pass it around, {i-1} bottle{"s" if i-1>1 else ""} of beer on the wall.')
		elif i==1:
			print(f'{i} bottle of beer on the wall, {i} bottle of beer. Take one down and pass it around, no more bottles of beer on the wall.')
		else:
			print(f'No more bottles of beer on the wall, no more bottles of beer. Go to the store and buy some more, {num} bottle{"s" if num>1 else ""} of beer on the wall.')
		





bottle_song(99)

File: 99_bottles.js
function bottleSong(num) {
  for (let i =num; i>=0; i--){
    if (i>1){
      console.log(`${i} bottles of beer on the wall, ${i} bottles of beer. Take one down and pass it around, ${i-1} bottle${i-1==1 ? "" : "s"} of beer on the wall.`)
    } else if (i==1) {
      console.log(`${i} bottle of beer on the wall, ${i} bottle of beer. Take one down and pass it around, no more bottles of beer on the wall.`)
    } else {
      console.log(`No more bottles of beer on the wall, no more bottles of beer. Go to the store and buy some more, ${num} bottle${num==1 ? "" : "s"} of beer on the wall.`)
    }
  }
};

bottleSong(99);
